src/main/java/duke/entity/TaskList.java

```diff
@@ -0,0 +1,35 @@
+package duke.entity;
+
+import duke.entity.task.Task;
+
+import java.util.ArrayList;
+import java.util.List;
+
```

Comment:
You may want to include JavaDoc comments to indicate what this class represents :)